ocaml/xapi/xapi_vbd_helpers.ml

```diff
@@ -169,12 +169,12 @@ let valid_operations ~expensive_sharing_checks ~__context record _ref' : table =
     if not vdi_record.Db_actions.vDI_managed
     then set_errors Api_errors.vdi_not_managed [ _ref ] all_ops;
 
-    let vdi_operations_besides_copy =
+    let operation_cannot_be_performed_live=
       List.exists
-        (fun (_, operation) -> operation <> `copy)
```

Comment:
I like that, too. Is it a problem that we are using polymorphic variants? These are open and we might never know the full list.